docs/backend/schema.ddl

```diff
@@ -44,59 +45,33 @@ Table GroupCanSee {
 
 
 ///////////////////////////////////////////////////
-// Physical Samples
+// Samples
 
-Table PhysicalSample {
+Table Sample {
   id INT [pk, increment]
   submitting_group_id INT [not null, ref: > Group.id]
-  private_identifier VARCHAR [not null]
+  private_identifier VARCHAR [not null]  // ideally this is the same as specimen_collector_sample_id noted below, but many DPHs will not be comfortable sharing these
 
   original_submission JSONB [not null]
 
   // This is the public identifier we assign to this sample.  The identifier assigned by the public repositories is
   // stored on the related Accession objects.
-  public_identifer VARCHAR [not null, unique]              // maps to specimen_collector_sample_id and isolate
+  public_identifer VARCHAR [not null, unique]              // maps to isolate, force format to pathogen/country/specimen_collector_sample_id/year, e.g., hCoV-19/USA/CZB-1234/2020
```

Comment:
Should maybe just be `USA/CZB-1234/2020` (without the `hCov-19` in front) since the prefix differs between GISAID and NCBI -- NCBI uses `SARS-CoV-2/human` as the prefix instead